R/check_post_tags.R

```diff
@@ -0,0 +1,18 @@
+# Get list of blog posts ----
+posts <- list.files("posts", recursive = TRUE, pattern = "*.qmd")
+
+# Function to extract tags from a post and check them against the allowed list ----
+check_post_tags <- function(post) {
+  allowable_tags <- c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")
```

Comment:
I would recomment defining the tags globally, then you can reference the tags in the "create_blogpost" script as well while having just one source of truth.